lib/LaTeXML/Engine/LaTeX.pool.ltxml

```diff
@@ -2772,11 +2772,16 @@ DefPrimitive('\DeclareTextSymbol DefToken {}{Number}', sub {
 '\expandafter\ifx\csname\cf@encoding\string' . $css . '\endcsname\relax\csname?\string' . $css . '\endcsname'
           . '\else\csname\cf@encoding\string' . $css . '\endcsname\fi'); }
     my $ecs = T_CS('\\' . $encoding . $css);
-    DefPrimitiveI($ecs, undef, FontDecode($code, $encoding));
+    DefPrimitiveI($ecs, undef, sub {
+        my ($glyph, $adjfont) = FontDecode($code, $encoding);
+        Box($glyph, $adjfont, undef, $cs); });
     return; });
 
-# hmmm... what needs doing here; basically it means use this encoding as the default for the symbol
-DefMacro('\DeclareTextSymbolDefault DefToken {}', '');    # '\DeclareTextSymbol{#1}{?}');
+DefPrimitive('\DeclareTextSymbolDefault DefToken {}', sub {
+    my ($stomach, $cs, $encoding) = @_;
+    $encoding = ToString(Expand($encoding));
```

Comment:
Interestingly, there's already an `Expanded` parameter type, but it's slightly special: requires braces and treats `\the` specially.  Perhaps it wants a clearer name, since "expanded" is so generic and would be tempting to use for what you suggest here.

Reply:
Right, what is the existing `Expanded` parameter type meant to be I wonder? I half-remember it being related to `\edef`, but that is the parameter `DefExpanded` nowadays.

In the spirit of the `XToken` parameter, maybe a name such as `XPlain` would be a good mnemonic for what I'm after? But maybe such a name is hard to "explain".

Reply:
Used in `\pdfstrcmp`, but I think `XGeneralText` is actually what should be used.  Need to do a bit more checking, and followup with another PR. Maybe take up your initial suggestion?